wolfcrypt/src/asn.c

```diff
@@ -6136,14 +6144,15 @@ int ExtractDate(const unsigned char* date, unsigned char format,
     }
 
     /* adjust tm_year, tm_mon */
-    if (GetTime(&certTime->tm_year, date, idx) != 0) return 0;
-    certTime->tm_year -= 1900;
-    if (GetTime(&certTime->tm_mon , date, idx) != 0) return 0;
-    certTime->tm_mon  -= 1;
-    if (GetTime(&certTime->tm_mday, date, idx) != 0) return 0;
-    if (GetTime(&certTime->tm_hour, date, idx) != 0) return 0;
-    if (GetTime(&certTime->tm_min , date, idx) != 0) return 0;
-    if (GetTime(&certTime->tm_sec , date, idx) != 0) return 0;
+    tm_year -= 1900;
+    tm_mon  -= 1;
```

Comment:
The month/time are adjusted here ... but I can can move them up to where the variables are defined if you like.